src/actions/MapActions.js

```diff
@@ -27,6 +27,7 @@ export const initializeMap = ({region}) => {
                 const sites = _.map(querySnapshot.docs, (doc, index) => {
                     let preparedSite = _.clone(doc.data());
                     preparedSite.id = doc.id;
+                    preparedSite.key = preparedSite.id;
```

Comment:
Several things (including the `FlatList`) expect unique keys for lists, and our most common list is our sites. Might as well add it as soon as we get the sites so we have it throughout the app.